### PR TITLE
hdac-ext-stream: rename variables - take 2

### PR DESCRIPTION
**Pull request by plbossart**
Now that we have all the DMA updates in the code, let's revisit the naming and cleanup the HDaudio code.

### CODE REVIEW

File: sound/hda/ext/hdac_ext_stream.c [hunk 1]
```diff
@@ -371,34 +372,34 @@ EXPORT_SYMBOL_GPL(snd_hdac_ext_stream_assign);
...
 
 	case HDAC_EXT_STREAM_TYPE_HOST:
 		spin_lock_irq(&bus->reg_lock);
-		if (stream->decoupled && !stream->link_locked)
-			snd_hdac_ext_stream_decouple_locked(bus, stream, false);
-		spin_unlock_irq(&bus->reg_lock);
-		snd_hdac_stream_release(&stream->hstream);
+		if (hext_stream->decoupled && !hext_stream->link_locked)
+			snd_hdac_ext_stream_decouple_locked(bus, hext_stream, false);
+		spin_lock_irq(&bus->reg_lock);
```
Comment (kv2019i) [inline sound/hda/ext/hdac_ext_stream.c]: @plbossart  What happened here, spon_unlock_req() turned to spin_lock_irq() ?

**Review by kv2019i — CHANGES_REQUESTED** [review]
New names look good. Patches are otherwise ready to go, but there's a unintentional-looking spinlock change in the first patch. With that fixed, this is good to go.

File: sound/hda/ext/hdac_ext_stream.c (hunk 1 above)
Comment (plbossart) [reply sound/hda/ext/hdac_ext_stream.c]: bad git merge I guess. Thanks for pointing this out.

Comment (plbossart) [reply sound/hda/ext/hdac_ext_stream.c]: fixed now.

**Review by kv2019i — APPROVED** [review]

**Review by RanderWang — APPROVED** [review]
LGTM

**Comment by plbossart** [comment]
@ujfalusi @ranj063 can you take a look?

**Pull request merged**